trino/trino.go

```diff
@@ -2029,7 +2029,7 @@ func parseNullTime(v string) (NullTime, error) {
 	var t time.Time
 	var err error
 	for _, layout := range timeLayouts {
-		t, err = time.ParseInLocation(layout, v, time.Local)
+		t, err = time.ParseInLocation(layout, v, time.UTC)
```

Comment:
Please also update tests.

Reply:
My bad, ran the tests on a different branch and sent it. Fixing this

Reply:
This is done